pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerImpl.java

```diff
@@ -448,7 +452,7 @@ public void sendAsync(Message<?> message, SendCallback callback) {
                         new PulsarClientException.InvalidMessageException(
                                 format("The producer %s of the topic %s sends a %s message with %d bytes that exceeds"
                                                 + " %d bytes",
-                        producerName, topic, compressedStr, compressedSize, ClientCnx.getMaxMessageSize()));
+                        producerName, topic, compressedStr, compressedSize, chunkMaxMessageSize));
```

Comment:
It seems that we should not change here.  If user set `chunkMaxMessageSize` but `isChunkingEnabled` is false, this error log is not accurate.